sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/perf/Scenarios/Deserialize.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging;
+using Azure.Test.Perf;
+using TestSchema;
+
+namespace Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.Perf.Scenarios
+{
+    public class Deserialize : SchemaRegistryAvroSerializerPerfTestBase
+    {
+        private readonly List<MessageContent> _serializedData = new();
+        public Deserialize(SizeCountOptions options) : base(options)
+        {
+            var employee = new Employee() { Name = "Caketown", Age = 42 };
```

Comment:
This is probably best done as part of `GlobalSetup` and used as a static set across parallel instances.

Reply:
GlobalSetup is an instance method, so I'm not sure how that would work. This can be done in a static constructor to share across instances, but I don't think it really matters.

Reply:
Global setup is called once across all instances of the tests before anything is run.   It's intended to be used similarly to a static constructor.